Formula/mp4v2.rb

```diff
@@ -17,6 +17,7 @@ class Mp4v2 < Formula
 
   conflicts_with "bento4",
     :because => "both install `mp4extract` and `mp4info` binaries"
+  patch :DATA
```

Comment:
Has this patch been submitted and accepted upstream?

We need a comment explaining the patch. The patch should also be submitted upstream, or at least reported, with an indication that it will be fixed.

Sorry for the delay on this review.